libs/community/langchain_community/embeddings/huggingface.py

```diff
@@ -167,6 +170,20 @@ def __init__(self, **kwargs: Any):
         except ImportError as e:
             raise ImportError("Dependencies for InstructorEmbedding not found.") from e
 
+        if "show_progress_bar" in self.encode_kwargs:
+            warn_deprecated(
+                since="0.2.5",
+                removal="0.4.0",
```

Comment:
Logistical question since this is my first contribution but I expect more - how does the removal actually happen as a part of 0.4.0; is there a backlog of these somehow generated when preparing for releases?

Reply:
We'll scan the code base prior to release of 0.4.0 for `warn_deprecated` and `@deprecated` usage to see what's targeted for removal